scm-ui/ui-syntaxhighlighting/src/worker/SyntaxHighlighter.worker.ts

```diff
@@ -91,7 +92,7 @@ const countChildrenAndApplyMarkers = (node: RefractorNode, markedTexts?: string[
   }
 
   if (newChildren.length > 0) {
-    const el = node as RefractorNode;
+    const el = node as unknown as RefractorElement;
```

Comment:
:)

Reply:
Its a necessary hack to get `tsc` happy, i know :'(